### DISCUSSION TITLE
cannot install torch==1.13.1 facing request timeout and getting killed

### DISCUSSION THREAD

**Discussion by sainadh812** [Xilinx/finn#988] [Q&A] [answered]
![image](https://github.com/Xilinx/finn/assets/58074126/e80fa634-c7c4-480b-8d7b-1ef18567c3f4)

how to resolved this issue , i clone using command  "git clone -b dev https://github.com/Xilinx/finn.git"

**Comment by auphelia** [comment, marked as answer]
Hi @sainadh812,
Does this problem still persist? It looks like a connectivity issue with your internet or maybe DNS issue that prevents installation of the dependencies?

**Reply by auphelia** [reply]
I am marking my message above as an answer, please feel free to unmark/reopen if you see the necessity for that.
